docs/asciidoc/virtual-graph.adoc

```diff
@@ -49,8 +49,48 @@ It's basic structure is: `{name:"Name",properties:{properties},nodes:[nodes],rel
 | apoc.graph.fromPaths([paths],'name',{properties}) | creates a virtual graph object for later processing
 | apoc.graph.fromDB('name',{properties}) | creates a virtual graph object for later processing
 | apoc.graph.fromCypher('statement',{params},'name',{properties}) | creates a virtual graph object for later processing
+| apoc.graph.inverseToTree({json},writeToGraph) | create a virtual or write to graph from a Json
```

Comment:
write to grpah in a config map
add abiltiy to specify fields for type and id
perhaps also id-field per type

we might have to consider more complex issues, e.g. containment-by-reference/value
and what happens if nested objects/arrays don't have type/id-information

see here for some examples/discussions https://neo4j.com/developer/neo4j-doc-manager/